youtube_dl/extractor/servus.py

```diff
@@ -5,26 +5,24 @@
 
 
 class ServusIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?servus\.com/(?:at|de)/p/[^/]+/(?P<id>AA-\w+|\d+-\d+)'
-    _TESTS = [{
-        'url': 'https://www.servus.com/de/p/Die-Gr%C3%BCnen-aus-Sicht-des-Volkes/AA-1T6VBU5PW1W12/',
-        'md5': '046dee641cda1c4cabe13baef3be2c1c',
+    _VALID_URL = r'https?://(?:www\.)?servus\.com/tv/videos/(?P<id>aa-\w+|\d+-\d+)'
+    _TEST = {
+        'url': 'https://www.servus.com/tv/videos/aa-1t6vbu5pw1w12/',
+        'md5': '3e1dd16775aa8d5cbef23628cfffc1f4',
         'info_dict': {
             'id': 'AA-1T6VBU5PW1W12',
             'ext': 'mp4',
-            'title': 'Die Grünen aus Volkssicht',
-            'description': 'md5:052b5da1cb2cd7d562ef1f19be5a5cba',
-            'thumbnail': r're:^https?://.*\.jpg$',
+            'title': 'Die Grünen aus Sicht des Volkes - Servus TV',
```

Comment:
` - Servus TV` should not be in title.

Reply:
But it is in the title:

AssertionError: Invalid value for field title, expected u'Die Gr\xfcnen aus Sicht des Volkes', got u'Die Gr\xfcnen aus Sicht des Volkes - Servus TV'

Reply:
Again: it's not part of the title and must not be in the title.

Reply:
Again: It is part of the title

Reply:
See here: <meta property="og:title" content="Die Grünen aus Sicht des Volkes - Servus TV" />

Reply:
`<meta property="og:title" content="Die Grünen aus Sicht des Volkes - Servus TV" />`

Reply:
I'm not going to argue. You either implement it as requested or it's not going to be merged.
![image](https://user-images.githubusercontent.com/1908898/53301615-8d11b880-3887-11e9-9815-620837af85fe.png)

Reply:
Look at the source of the Website, the title has servus tv in there

Reply:
I will change title to regex, but i cannot push this